osquery/tables/system/windows/os_version.cpp

```diff
@@ -48,12 +49,25 @@ QueryData genOSVersion(QueryContext& context) {
   wmiResults[0].GetString("InstallDate", cimInstallDate);
   r["install_date"] = BIGINT(cimDatetimeToUnixtime(cimInstallDate));
 
+  QueryData regResults;
+  queryKey(
+      "HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion",
+      regResults);
+
+  std::string updateBuildRevision{""};
+  for (const auto& aKey : regResults) {
+    if (aKey.at("name") == "UBR") {
+      updateBuildRevision = "." + aKey.at("data");
+      break;
+    }
+  }
+
   wmiResults[0].GetString("Version", version_string);
   auto version = osquery::split(version_string, ".");
 
   switch (version.size()) {
   case 3:
-    r["build"] = SQL_TEXT(version[2]);
+    r["build"] = SQL_TEXT(version[2] + updateBuildRevision);
```

Comment:
Do you think this better added here, or as a separate column? I don't know what expected results are. I'll poll some folks

Reply:
Chatting with people, it sounds like there's a strong bias to make this a new column.

Reply:
probably called "revision"

Reply:
Thanks for taking time out to reivew, agree with your comments,  will implement the necessary changes and get back

Reply:
> probably called "revision"

![image](https://github.com/osquery/osquery/assets/38827245/cfc50ed2-025f-4dd1-b363-8189ef99e361)
please let me know your thoughts on this or do you want to me leave the existing values as is and only provide the new revision column